cairo/ethereum/cancun/state.cairo

```diff
@@ -64,3 +82,96 @@ struct StateStruct {
 struct State {
     value: StateStruct*,
 }
+
+using OptionalAccount = Account;
+func get_account_optional{poseidon_ptr: PoseidonBuiltin*, state: State}(
+    address: Address
+) -> OptionalAccount {
+    let trie = state.value._main_trie;
+    with trie {
+        let account = trie_get_TrieAddressAccount(address);
+    }
+
+    return account;
+}
+
+func get_account{poseidon_ptr: PoseidonBuiltin*, state: State}(address: Address) -> Account {
+    let account = get_account_optional{state=state}(address);
+
+    if (cast(account.value, felt) == 0) {
+        let empty_account = EMPTY_ACCOUNT();
+        return empty_account;
+    }
+
+    return account;
+}
+
+func get_storage{poseidon_ptr: PoseidonBuiltin*, state: State}(
```

Comment:
Why get_storage is in this PR ?

Reply:
i was grouping smaller issues

Reply:
you might want to update the description and title then